web/themes/custom/lex/templates/pages/page.html.twig

```diff
@@ -14,24 +14,24 @@
 #}
 
 {% block page_alerts %}
-  {{ page.alerts }}
+	{{ page.alerts }}
```

Comment:
Idk why 2 space indents are looking like 6 here. 🤷

Reply:
Yeah I was having that issue too. Weird.